tests/page-objects/login/login.po.js

```diff
@@ -30,6 +30,7 @@ module.exports = {
     changeLocale(locale);
     getLoginButton().click();
     browser.waitForAngular();
+    browser.sleep(200);//seems to be failing here...maybe local browser slow
```

Comment:
We'll want to get to the bottom of this one...

Reply:
removed - it seems stable enough on travis...something slow on my machine -